src/data/reducers/index.js

```diff
@@ -10,6 +10,8 @@ const studioDetails = () => studioContext;
 const rootReducer = combineReducers({
```

Comment:
I could very well misunderstand how redux works, but I thought you had to have one "reducer" function that redux calls. `combineReducer` allows you to split that into multiple separate functions and still have that single top-level function.

Reply:
According to [this](https://redux.js.org/docs/api/combineReducers.html#combinereducersreducers) documentation (second paragraph), you pass a single function to the [createStore](https://redux.js.org/docs/api/createStore.html) function. The combineReducers call here creates a single "root reducer" that you pass to the createStore function, so it is necessary to group together all the reducers in this way. That's the purpose of the combineReducers function - to be able to split and slice your store state. Since they're all sharing the same store, we have to group them together, even if they operate on different parts of the store, I believe.